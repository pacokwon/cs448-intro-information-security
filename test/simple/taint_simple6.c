#include "homework.h"

int main() {
	int x = source();
	int result = 0;

	while(result<5)
	{
            result += sanitizer(x);
	}

        sink(result);
	sink(x);

	return 0;
}
