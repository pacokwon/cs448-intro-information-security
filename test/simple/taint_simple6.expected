Potential Tainted-flow @ simple/taint_simple6.c:main:13:2 (call void @sink(i32 %call), !dbg !23)
